compiler/rustc_hir_analysis/Cargo.toml

```diff
@@ -13,6 +13,7 @@ itertools = "0.12"
 rustc_arena = { path = "../rustc_arena" }
 rustc_ast = { path = "../rustc_ast" }
 rustc_attr = { path = "../rustc_attr" }
+rustc_codegen_ssa = { path = "../rustc_codegen_ssa" }
```

Comment:
It doesn't seem right for type checking to depend on codegen logic

Reply:
Agreed - the reason for this dependency is that parsing the target feature attribute currently sits in the codegen sub-crate (specifically `rustc_codegen_ssa::target_features::from_target_feature` afaiu), and I did not want to figure out a better home for it yet - especially if the design here is completely broken :-)